lib/src/http.dart

```diff
@@ -68,7 +68,7 @@ class _PubHttpClient extends http.BaseClient {
       streamedResponse = await _inner.send(request);
     } on SocketException catch (error, stackTrace) {
       // Work around issue 23008.
-      if (stackTrace == null) stackTrace = new Chain.current();
+      final defaultedStackTrace = stackTrace ?? new Chain.current();
```

Comment:
Nit: `var`

I'd call the caught stack trace `stackTraceOrNull` and call this `stackTrace`, since this is used more frequently.

Reply:
Done